composer.json

```diff
@@ -10,13 +10,17 @@
             "email": "mailchimp@ebizmarts-desk.zendesk.com"
         }
     ],
-    "require": {
-        "magento-hackathon/magento-composer-installer": "*"
+    "suggest": {
+        "magento-hackathon/magento-composer-installer": "Map/copy module files with your Magento installation"
+    },
+    "require-dev": {
+        "magento-ecg/coding-standard": "^2.0",
+        "magento/marketplace-eqp": "^1.0"
     },
     "repositories": [
         {
-            "type": "composer",
-            "url": "https://packages.firegento.com"
+            "type": "vcs",
+            "url": "https://github.com/magento/marketplace-eqp.git"
```

Comment:
I believe that the lack of a GitHub token is the problem.
Can I get you to generate a GitHub token ([link to generate token](https://github.com/settings/tokens/new?scopes=repo)) with access to the repo scopes and pass it as a [repository environment variable](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings) named `GITHUB_TOKEN`?